truthy-falsy.js

```diff
@@ -0,0 +1,117 @@
+// 1
+const isEmployed = true;
+const isOnVacation = false;
+
+function setAlarm(isEmployed, isOnVacation){
+  return isEmployed && !isOnVacation;
+}
+console.log(setAlarm(isEmployed, isOnVacation));
+
+// 2
+function closeCompare(numberA, numberB, numbersMargin){
+  if (numbersMargin === undefined) numbersMargin = 0;
```

Comment:
Always use `{` and `}` when writing `if` statements.

Reply:
Never change the function arguments like that
```
numbersMargin = 0
```

Reply:
You can use default parameters instead
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters